components/style/properties/properties.mako.rs

```diff
@@ -2588,6 +2591,20 @@ pub fn apply_declarations<'a, F, I>(device: &Device,
     style.build()
 }
 
+/// See StyleAdjuster::adjust_for_auto_minsize.
+% if product == "servo":
+pub fn adjust_auto_minsize(style: &mut StyleBuilder, is_item: bool) {
+    % for direction in ["width", "height"]:
+        // Like calling to_computed_value, which wouldn't type check.
+        if style.get_position().clone_min_${direction}() == computed::LengthOrPercentageOrAuto::Auto
+            && !is_item {
+                style.mutate_position().set_min_${direction}(
+                    computed::LengthOrPercentageOrAuto::Length(Au(0))
```

Comment:
Sorry, the comment above doesn't represent well the dependence on overflow, though it's obvious reading Gecko's source.

The following test-case:

```html
<!doctype html>
<style>
.item {
  min-width: auto;
  min-height: auto;
  overflow: hidden;
}
</style>
<div id="log"></div>
<div style="display: flex">
  <div class="item"></div>
</div>
<script>
var item = document.querySelector('.item');
alert(getComputedStyle(item).minHeight)
alert(getComputedStyle(item).minWidth)
</script>
```

Alerts `0px`, `0px` on FF, but `auto`, `auto` on Blink. I'm looking for the specs to see who's the one right here.